packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -115,8 +115,6 @@ async function fetchBlobsFromSnapshotTree(
     tree: ISnapshotTree,
     prefix: string = "/",
     perCommitBlobIdMap?: Map<string, number>): Promise<IFetchedData[]> {
```

Comment:
perCommitBlobIdMap and const commit below should be renamed. The difference now is subtree versus top-level.